features/atomic/atomic_mount_unmount.feature

```diff
@@ -0,0 +1,49 @@
+Feature: Atomic mount and unmount test for new upgrade tree
+    Describes tests for mount/umount to new ostree
+
+Background: Atomic hosts are discovered
+      Given "cihosts" hosts from dynamic inventory
+
+  Scenario: 1. Host provisioned and subscribed
+      Given "cihosts" host
+       When "cihosts" host is auto-subscribed to "stage"
+       Then subscription status is ok on "cihosts"
+        and "1" entitlement is consumed on "cihosts"
+
+  Scenario: 2. 'atomic host upgrade' is successful
+      Given there is "1" atomic host tree deployed
+       When atomic host upgrade is successful
+       Then wait "60" seconds for "cihosts" to reboot
+       Then there is "2" atomic host tree deployed
+
+  Scenario: 3. docker run busybox with detach mode
+      Given "docker" is already installed on "cihosts"
+        and "docker" is already running on "cihosts"
```

Comment:
Since the previous scenario performs a `atomic upgrade`, I think it is safe to assume that the `docker` package is installed since RHEL Atomic, Fedora Atomic, and CentOS Atomic all have it installed by default.